util/chplenv/chpl_gmp.py

```diff
@@ -12,10 +12,10 @@ def get():
     gmp_val = overrides.get('CHPL_GMP')
     if not gmp_val:
         target_compiler = chpl_compiler.get('target')
-        if target_compiler == 'cray-prgenv-cray':
+        target_platform = chpl_platform.get('target')
+        if target_compiler == 'cray-prgenv-cray' and target_platform.startswith('cray-x'):
```

Comment:
Can you make an issue about dropping this for cray-x? (and feel free to assign to me)

Reply:
https://github.com/Cray/chapel-private/issues/1473

I assigned it to both of us.  I have a slight preference for doing it myself, because I think that will make it easier to cherry-pick my EX-specific changes here back into the main line.  But either is ultimately okay.